app/uk/gov/hmrc/formpproxy/repositories/CisFormpRepository.scala

```diff
@@ -152,6 +153,9 @@ class CisFormpRepository @Inject()(@NamedDatabase("cis") db: Database)(implicit
   private val SqlGetSchemeVersion =
     "select version from scheme where instance_id = ?"
 
+  private val CallGetSchemeEmail =
+    "{ call IntGetSchemeSp(?, ?) }"
```

Comment:
Is this a stored proc? where is it in the database?

Reply:
that's correct @yanshengzhangHMRC

Reply:
I discussed with Waigwa who confirmed that the naming convention difference is not an issue, as it worked locally for him.